### ISSUE TITLE
Event when user clicks 'next' or 'prev'

### ISSUE THREAD

**Issue by Fischbrot** [UniversalDataTool/react-image-annotate#131]
Is it possible to detect when a user click next or prev?

I want to add a progressbar on my project, but i need to know when a user clicks next or prev.

Else i'd have to only put a single image in each instance and then reload the states with a custom next/prev button.

**Comment by paulkr** [comment]
Yeah, I was having the same issue with the latest version. Using 1.7.10 works for me though!

**Comment by Cerealplacebo5** [comment]
In the 1.7.10 version, there are two props (onNextImage and onPrevImage), for functions that will be triggered by the next and prev button.
